Description: Fix for Hardening
 Modify the stable Makefile to enable hardening
Author: Daniel Lintott <daniel@serverb.co.uk>
---
Index: dynamips/stable/Makefile
===================================================================
--- dynamips.orig/stable/Makefile	2013-10-07 17:26:34.347360332 +0100
+++ dynamips/stable/Makefile	2013-10-07 17:32:52.450606931 +0100
@@ -72,7 +72,7 @@
 #   PCAP_LIB=-lpcap
    OSNAME=Linux
    CFLAGS+=-I/usr/include -I. $(PTHREAD_CFLAGS)
-   LIBS=-L/usr/lib -L. -ldl -lrt -luuid -lelf $(PTHREAD_LIBS)
+   LIBS=-L/usr/lib -L. -ldl -lrt -luuid -lelf $(PTHREAD_LIBS) $(LDFLAGS)
    DESTDIR=/usr
 else
 ifeq ($(shell uname -s), Darwin)
@@ -282,7 +282,7 @@
 
 nvram_export$(BIN_EXT): nvram_export.o fs_nvram.o
 	@echo "Linking $@"
-	@$(CC) -Wall $(CFLAGS) $(INCLUDE) -o $@ nvram_export.o fs_nvram.o
+	@$(CC) -Wall $(CFLAGS) $(INCLUDE) $(LDFLAGS) -o $@ nvram_export.o fs_nvram.o
 
 install: $(PROG) nvram_export$(BIN_EXT)
 	@echo "Installing"
@@ -321,7 +321,7 @@
 
 .c.o:
 	@echo "Compiling $<"
-	@$(CC) $(CFLAGS) $(INCLUDE) -c -o $*.o $<
+	@$(CC) $(CPPFLAGS) $(CFLAGS) $(INCLUDE) -c -o $*.o $<
 
 .l.c:
 	$(LEX) -o$*.c $<
